### PR TITLE
push dev template to vagrant-hashistack-template-dev

### LINKED ISSUE #87: Automatic update of sub-repo for templating
#86 
#83

### PR DESCRIPTION
**Pull request by fredrikhgrelland**
close #87

### CODE REVIEW

**Comment by fredrikhgrelland** [comment]
POC worked: https://github.com/fredrikhgrelland/vagrant-hashistack-template-dev
Will move it into the `release`job

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@ jobs:
     if: github.event.action == 'closed' && github.event.pull_request.merged == true
     needs: release-prerequisites
 
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```
Comment (zhenik) [inline .github/workflows/build.yml:104]: Could you elaborate why `ubuntu-latest`?

File: template/dev/README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Development template for `fredrikhgrelland/hashistack`
 
 This template can be used as a base image for developing services on the hashistack.
+If you found this in `fredrikhgrelland/vagrant-hashistack`, you may be interested in this separate [repository](https://github.com/fredrikhgrelland/vagrant-hashistack-template-dev)
```
Comment (zhenik) [inline template/dev/README.md:4]:
```suggestion
If you found this in `fredrikhgrelland/vagrant-hashistack`, you may be interested in this separate repository [vagrant-hashistack-template-dev](https://github.com/fredrikhgrelland/vagrant-hashistack-template-dev)
```

File: .github/workflows/build.yml (hunk 1 above)
Comment (fredrikhgrelland) [reply .github/workflows/build.yml:104]: Yes. The plugin only works on linux and none of the other parts need to run on mac. Only vagrant builds/test need virtualbox/vagrant

**Pull request merged**